internal/rukpak/source/containers_image.go

```diff
@@ -45,6 +48,22 @@ func (i *ContainersImageRegistry) Unpack(ctx context.Context, bundle *BundleSour
 	if err != nil {
 		return nil, err
 	}
+
+	//////////////////////////////////////////////////////
+	//
+	// Verify that registries configuration is up-to-date
+	//
+	//////////////////////////////////////////////////////
+	data, err := os.ReadFile(srcCtx.SystemRegistriesConfPath)
+	if err != nil {
+		return nil, fmt.Errorf("could not read registries configuration file, error: %w", err)
+	}
+	registriesConfHash := sha256.Sum256(data)
```

Comment:
It would be more memory efficient to `os.Open` + `defer file.Close(), and then `io.Copy` from the file into an instance of the hash object,. If you choose to make that change, I'd also suggest extracting that logic into a separate function since it is slightly more complex.